mod.ts

```diff
@@ -7,7 +7,7 @@ import themes from "./steps/themes.ts";
 import save from "./steps/save.ts";
 import plugins from "./steps/plugins.ts";
 import success from "./steps/success.ts";
-import git from "./steps/git.ts";
+import git from "./steps/gitignore.ts";
```

Comment:
I called it `git.ts` because can be used for all git stuff. For example, in the future we can include `.gitattributes`, etc.